.github/renovate.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    ":semanticCommitsDisabled"
+  ],
+  "labels": ["dependencies"],
+  "rebaseWhen": "conflicted",
+  "ignoreDeps": [
+    "org.gretty",
+    "org.kohsuke.stapler:stapler-jelly",
+    "org.kohsuke.stapler:stapler",
+    "eclipse-temurin"
+  ],
```

Comment:
I assume all these ignores can be removed now? prob from when stuck on java 8

Reply:
Gretty 4 depends on Jetty 11, we're using Jetty 10 still. eclipse-temurin ensures we stay on Java 17 and stapler I don't recall anything in particular.